src/Components/Sorting/Sorting.js

```diff
@@ -0,0 +1,144 @@
+import React from "react";
+import CHARACTER from "./SortingData";
+import SortingChar from "./SortingChar";
+import "./Sorting.scss";
```

Comment:
```suggestion
import React from "react";
import SortingChar from "./SortingChar";
import "./Sorting.scss";
import CHARACTER from "./SortingData";
```
라이브러리 -> 컴포넌트 -> css, 기타 순서로 import 해주세요!